releasenotes/notes/Add_support_for_BackendV2_in_instruction_durations_from_backend-9ce722879c870377.yaml

```diff
@@ -0,0 +1,22 @@
+---
+features:
+  - |
+    This enables the support for :class:`.BackendV2` in :meth:`.from_backend`
+    of :class:`.InstructionDurations`.
+    
+    Users can have an object of :class:`.InstructionDurations` using :class:`.BackendV2`
+    from :meth:`.from_backend` with followig code.
+    
+    .. code-block:: python
+
+      from qiskit.providers.fake_provider import FakePerth
+      from qiskit.transpiler import InstructionDurations
+      backendV2 = FakePerth()
+      
+      inst_dur = InstructionDurations.from_backend(backendV2)
+
+upgrade:
+  - |
+    Given :code:`dt` and :code:`dtm` for a :class:`.BackendV1` or :class:`.BackendV2` are unequal,
+    :meth:`~.InstructionDurations.from_backend` does not raise any error.
```

Comment:
Is there any change in code behavior with this PR? `dtm` is acquisition sampling rate and doesn't impact the instruction duration.

Reply:
@nkanazawa1989 , 
According to, https://github.com/Qiskit/qiskit/blob/828ad4d623a2b328739691f66bb67facba685971/qiskit/transpiler/instruction_durations.py#L76

I should get `TranspilerError` if `dt`!=`dtm`, I just checked it, and found it doesn't.
So, I added this line in release note and added test `test_works_unequal_dt_dtm` to make sure this case is covered :)

Reply:
The document should be wrong. See p.33 of https://arxiv.org/abs/1809.03452. dtm defines the sampling rate of digitizer, which can be independent of dt defines the sampling rate of AWG.

Reply:
I think this upgrade note is not necessary because `TranspilerError` was not actually raised (just accidentally documented).